bigquery/snippets/snippet.go

```diff
@@ -817,6 +825,24 @@ func updateTableChangeCMEK(client *bigquery.Client, datasetID, tableID string) e
 	return nil
 }
 
+func updateModelDescription(client *bigquery.Client, datasetID, modelID string) error {
+	ctx := context.Background()
+	// [START bigquery_update_model_description]
+	model := client.Dataset(datasetID).Model(modelID)
+	oldMeta, err := model.Metadata(ctx)
+	if err != nil {
+		return err
```

Comment:
Might want to add additional info to the error. https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#return-errors

Reply:
Done